src/server/pfs/s3/object.go

```diff
@@ -104,7 +104,7 @@ func (c *controller) CopyObject(r *http.Request, srcBucketName, srcFile string,
 		return "", s2.NotImplementedError(r)
 	}
 
-	if err = pc.CopyFile(srcBucket.Repo, srcBucket.Commit, srcFile, destBucket.Repo, destBucket.Commit, destFile); err != nil {
+	if err = pc.CopyFile(destBucket.Repo, destBucket.Commit, destFile, srcBucket.Repo, srcBucket.Commit, srcFile); err != nil {
```

Comment:
How did this not break something?

Reply:
Not sure I understand the question. The parameters for copy file are now (dst, src) rather than (src, dst) which means the arguments needed to be shuffled around.